### PR TITLE
refactor: manual tool documentation & QOL

### PR DESCRIPTION
**Pull request by JeremyTubongbanua**
**- What I did**

- (Documentation) fixed paths in the manual tool documentation
- --rm flag for the manual tool `run.sh` script
- `chmod +x clean-local.sh` for manual tool
- added more examples to usage output

**- How I did it**

- edited README.mds, run.sh, and clean-local.sh

**- How to verify it**

- All changes are in `test/` directory (no functional changes to SSH No ports)

**- Description for the changelog**

Documentation and QOL changes

### CODE REVIEW

File: tests/end2end_tests/tools/manual-tool/README.md [hunk 1]
```diff
@@ -95,7 +99,7 @@ Example: build and run a container to run sshnp on a particular release
 
 ```
 cd release
-sudo docker-compose build --build-arg release=3.3.0
+sudo docker-compose build --build-arg release=v3.3.0
```
Comment (cpswan) [inline tests/end2end_tests/tools/manual-tool/README.md:102]: Should we have specific release version numbers here?

File: tests/end2end_tests/tools/manual-tool/run.sh [hunk 2]
```diff
@@ -16,7 +17,9 @@ usage() {
     echo ""
     echo "  example: $0 -t sshnp -b trunk"
     echo "  example: $0 -t sshnpd -l"
-    echo "  example: $0 -t sshrvd -r 3.3.0"
+    echo "  example: $0 -t sshrvd -r v3.3.0"
```
Comment (cpswan) [inline tests/end2end_tests/tools/manual-tool/run.sh:20]: Again, hard coded version.

**Review by cpswan — APPROVED** [review]
I'm approving so the improvements here can be merged.

But... it can be made better...

Rather than hard coding version numbers we can grab the latest release version:

```
VERSION=$(curl -s "https://api.github.com/repos/atsign-foundation/sshnoports/releases/latest" | grep -Po '"tag_name": "v\K.*?(?=")')
```

**Comment by JeremyTubongbanua** [comment]
> I'm approving so the improvements here can be merged.
> 
> But... it can be made better...
> 
> Rather than hard coding version numbers we can grab the latest release version:
> 
> ```
> VERSION=$(curl -s "https://api.github.com/repos/atsign-foundation/sshnoports/releases/latest" | grep -Po '"tag_name": "v\K.*?(?=")')
> ```

The image grabs the latest version unless specified

```
  -r|--release [release] - build from a sshnoports release, latest release by default
```

**Pull request merged**